docs/en/guides/operational/gitops-with-amg/gitops-with-amg.md

```diff
@@ -0,0 +1,66 @@
+# Using GitOps and Grafana Operator with Amazon Managed Grafana
+
+## How to use this guide
+
+This Observability best practices guide is meant for developers and architects who want to understand how to use [grafana-operator](https://github.com/grafana-operator/grafana-operator#:~:text=The%20grafana%2Doperator%20is%20a,an%20easy%20and%20scalable%20way.) as a Kubernetes operator on your Amazon EKS cluster to create and manage the lifecycle of Grafana resources and Grafana dashboards in Amazon Managed Grafana in a Kubernetes native way.
+
+## Introduction
+
+Customers use Grafana as an observability platform for open source analytics and monitoring solution. We have seen customers running their workloads in Amazon EKS want to shift their focus towards workload gravity and rely on Kubernetes-native controllers to deploy and manage the lifecycle of external resources such as Cloud resources. We have seen customers installing [AWS Controllers for Kubernetes (ACK)](https://aws-controllers-k8s.github.io/community/docs/community/overview/) to create, deploy and manage AWS services. Many customers these days opt to offload the Prometheus and Grafana implementations to managed services and in case of AWS these services are [Amazon Managed Service for Prometheus](https://docs.aws.amazon.com/prometheus/?icmpid=docs_homepage_mgmtgov) and [Amazon Managed Grafana](https://docs.aws.amazon.com/grafana/?icmpid=docs_homepage_mgmtgov) for monitoring their workloads.
+
+One common challenge customers face while using Grafana is, in creating and managing the lifecycle of Grafana resources and Grafana dashboards in external Grafana instances such as Amazon Managed Grafana from their Kubernetes cluster. Customers face  challenges in finding ways to completely automate and manage infrastructure and application deployment of their whole system using Git based workflows which also includes creating of Grafana resources in Amazon Managed Grafana. In this Observability best practices guide, we will focus on the following topics:
+
+* Introduction on Grafana Operator - A Kubernetes operator to manage external Grafana instances from your Kubernetes cluster 
+* Introduction to GitOps - Automated mechanisms to create and manage your infrastructure using Git based workflows
+* Using Grafana Operator on Amazon EKS to manage Amazon Managed Grafana
+* Using GitOps with Flux on Amazon EKS to manage Amazon Managed Grafana
```

Comment:
"manage resources in Amazon Managed Grafana"

Reply:
Fixed.